src/data/community-events.ts

```diff
@@ -0,0 +1,395 @@
+import { CommunityConference } from "@/lib/types"
+
+const CommunityConferences: CommunityConference[] = [
```

Comment:
@corwintines Given this isn't a component, we should lowercase this variable name